opal-core-api/src/main/java/org/obiba/opal/core/domain/VCFSamplesMapping.java

```diff
@@ -0,0 +1,101 @@
+package org.obiba.opal.core.domain;
+
+import javax.validation.constraints.NotNull;
+
+public class VCFSamplesMapping {
+    @NotNull
+    private String projectName;
+
+    @NotNull
+    private String tableName;
```

Comment:
This is not a table name, this is a tableReference, i.e. the fully qualified name of a table: \<datasource>.\<table>